metis-dereference/metis-dereference-common/src/main/java/eu/europeana/metis/dereference/Vocabulary.java

```diff
@@ -78,15 +84,38 @@ public void setUris(Collection<String> uris) {
     this.uris = new HashSet<>(uris);
   }
 
+  /**
+   * Getter.
+   *
+   * @return The suffix.
+   * @deprecated Will be removed.
+   */
   @XmlElement
+  @Deprecated (forRemoval = true)
```

Comment:
todo comment with follow up ticket for removal?

Reply:
OK. Added in various places.